src/main/java/com/regnosys/testing/ExpectationUtil.java

```diff
@@ -0,0 +1,110 @@
+package com.regnosys.testing;
```

Comment:
Is this a copy of `com.regnosys.granite.ingestor.ExpectationUtil`?  Are you going to delete the other one?

Reply:
updated